src/components/common/Typography/Typography.css

```diff
@@ -46,6 +46,12 @@
 .Typography.Typography--md {
   font-size: 1.5rem;
 }
+.Typography.Typography--xxlg {
+  font-size: 2.5rem;
+}
 .Typography.Typography--primary {
   color: var(--color-primary) !important;
```

Comment:
Avoid adding important you increase the specificity i know sometimes it is quicker to just add important and get things done quick, i also do it but try increasing the specificity